Tools-Override/resources.targets

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is the intent to always override this or is it to move this update into BuildTools?